src/main/resources/graphql/service/query.graphqls

```diff
@@ -27,7 +27,7 @@ type Query {
     """
     Retrieve progress for multiple chapters
     """
-    progressByChapterIds(chapterIds: [UUID!]!): [CompositeProgressInformation!]!
+    _internal_noauth_progressByChapterIds(chapterIds: [UUID!]!): [CompositeProgressInformation!]!
```

Comment:
are you sure this isn't also used by the frontend? for the progress rings for the chapters?

Reply:
bro Tim did this ~2days ago, I'm currently implementing the gateway integration, if they use it they definetly shouldnt and I doubt they are this fast

Reply:
ah yea right. I thought tim only modified it to also return the total number of contents and so on and we had the query already before that.

What are the frontend then using for the chapter progress display?